src/main/surfrdMod.F90

```diff
@@ -863,14 +863,28 @@ subroutine surfrd_veg_all(begg, endg, ncid, ns, actual_numcft)
        ! Check sum of vegetation adds to 1
        call check_sums_equal_1(wt_cft, begg, 'wt_cft', subname)
     end if
-    ! Call collapse_crop_types:
+    ! Call collapse_crop_types: allows need to maintain only 78-pft input data
     ! For use_crop = .false. collapsing 78->16 pfts or 16->16 or some new
     !    configuration
     ! For use_crop = .true. most likely collapsing 78 to the list of crops for
     !    which the CLM includes parameterizations
     ! The call collapse_crop_types also appears in subroutine dyncrop_interp
-    call collapse_crop_types(wt_cft(begg:endg, :), fert_cft(begg:endg, :), cft_size, begg, endg, verbose=.true.)
-
+    call collapse_crop_types(wt_cft(begg:endg,:), fert_cft(begg:endg,:), cft_size, begg, endg, verbose=.true.)
+
+    ! Collapse crop variables as needed
+    ! The call to collapse_crop_var also appears in subroutine dyncrop_interp
+    ! - fert_cft TODO Is this call redundant because it simply sets the crop
+    !                 variable to 0 where is_pft_known_to_model = .false.?
```

Comment:
Same comment in dyncrop...
I kept this new subroutine, even though now it's only tangentially relevant because @billsacks said it would be helpful for crop variables. We can discuss whether to keep.